internal/lockclient/cache/lru_cache.go

```diff
@@ -213,9 +213,13 @@ func (lru *LRUCache) deleteElementFromMap(key interface{}) error {
 	}
 	return nil
 }
+
+// printMap prints the LRU map and is concurrency safe.
 func (lru *LRUCache) printMap() {
+	lru.mu.Lock()
```

Comment:
Adding a new datatype would be an overload. I dont think this competes in anyway because this is done while debugging and we dont need a whole new variable for that.